synapse/lib/lmdblayer.py

```diff
@@ -0,0 +1,418 @@
+'''
+The layer library contains the base Layer object and helpers used for
+cortex construction.
+'''
+import os
+import lmdb
+import regex
```

Comment:
style nit - we've generally been keeping third party modules after standard lib modules

Reply:
ok